chapters/operatorsandexpressions.tex

```diff
@@ -612,7 +617,7 @@ \subsection{Event Triggering Mathematical Functions}\label{event-triggering-math
 E.g., \lstinline!noEvent(integer(v))!.
 \end{nonnormative}
 
-\begin{operatordefinition}[div]
+\begin{functiondefinition}[div]
```

Comment:
As explained above, I don't think `div` is a function.  (Similarly for the other operators in this section.)

Reply:
To me the annotation `GenerateEvents` should allow you to write a function such as `div` in Modelica.

Reply:
This has been addressed.  Resolving.